oxd-server/src/main/java/org/gluu/oxd/server/op/InitiateThirdPartyLoginOperation.java

```diff
@@ -0,0 +1,40 @@
+package org.gluu.oxd.server.op;
+
+import com.google.inject.Injector;
+import org.gluu.oxd.common.Command;
+import org.gluu.oxd.common.params.ThirdPartyLoginParams;
+import org.gluu.oxd.common.response.GetAuthorizationUrlResponse;
+import org.gluu.oxd.common.response.IOpResponse;
+import org.gluu.oxd.server.Utils;
+import org.gluu.oxd.server.service.Rp;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class InitiateThirdPartyLoginOperation extends BaseOperation<ThirdPartyLoginParams> {
+
+    private static final Logger LOG = LoggerFactory.getLogger(InitiateThirdPartyLoginOperation.class);
+
+    protected InitiateThirdPartyLoginOperation(Command command, Injector injector) {
+        super(command, injector, ThirdPartyLoginParams.class);
+    }
+
+    @Override
+    public IOpResponse execute(ThirdPartyLoginParams params) throws Exception {
```

Comment:
Here we must validate `target_link_uri `.
Spec: 
```
RPs MUST verify the value of the target_link_uri to prevent being used as an open redirector to external sites.
```
Also unfortunately in current state oxd can't perform redirect after authorization. We will be able to improve it in 5.0 (later) after we implement this:
https://github.com/GluuFederation/oxd/issues/462

Anyway, lets perform validation since we are implementing it.

Reply:
added validation for `target_link_uri `.